triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/RunnerMain.scala

```diff
@@ -68,12 +69,20 @@ object RunnerMain {
         val sequencer = new AkkaExecutionSequencerPool("TriggerRunnerPool")(system)
         implicit val ec: ExecutionContext = system.dispatcher
 
+        val tokenHolder = config.accessTokenFile.map(new TokenHolder(_))
+        println(tokenHolder.flatMap(_.token))
```

Comment:
`println`? Are you sure?

Reply:
Oops, that was just leftover, thanks! Removed it now.